build.gradle

```diff
@@ -182,6 +182,11 @@ subprojects {
 
   task docsJar(dependsOn: javadocJar)
 
+  javadoc {
+    options.charSet = 'UTF-8'
+    options.encoding = 'UTF-8'
```

Comment:
Thank you so much.
I will set options `charSet`, `docsEncoding` and `encoding`.